src/server/config/auth.js

```diff
@@ -1,11 +1,29 @@
-export function validateAccessTokenFacebook(accessToken) {
-  ///TODO : Validiate accesstoken from facebook API server.
+import request from 'request';
 
-  return false;
+export function crawlByAccessTokenFacebook(accessToken, callback) {
+  // Crawl user data from facebook by access token.
+  request.get('https://graph.facebook.com/me?fields=name,email,locale,timezone,verified&access_token='
```

Comment:
Put the reference that prescribes this API. Facebook (and any API providers) often change their signatures which is always pain for API consumers (us). If we can declare these API URIs as const up top (so it's easy to spot), it'd be even better.